src/main.c

```diff
@@ -106,6 +107,13 @@ int main(int argc, char** argv)
     /* Register tests as required, then call the UT-main to support switches and triggering */
     UT_init( argc, argv );
 
+    if ( test_utils_parseconfig() == -1 )
```

Comment:
don't call things test_utils... 

Call it a real name,. and since this is part of main.c choices, it should be in the main.c file.

Reply:
Please review -> https://github.com/rdkcentral/ut-core/wiki/Standards:-Principles-from-Code-Complete

Reply:
updated to read directly from kvp